package.json

```diff
@@ -16,7 +16,7 @@
   "homepage": "https://github.com/autoricardo/carforyou-phraseapp-pkg#readme",
   "devDependencies": {
     "@carforyou/eslint-config": "3.1.7",
-    "@pika/pack": "^0.5.0",
+    "@pika/pack": "^0.3.7",
```

Comment:
what's the motivation for the downgrade here? the build scripts fail because of it

Reply:
that's changed after `npm audit fix --forces`, there was some vulnerabilities and this is the reason why it's downgraded

Reply:
I see. that's what the --force does. I would not downgrade pika though. let's stick to what the non-forced fix can do and wait for a fix in pika. these are build-time dependencies, so they would not be a risk for a production deployment